docs/devel/testing.md

```diff
@@ -73,32 +77,59 @@ is [table driven testing](https://github.com/golang/go/wiki/TableDrivenTests)
 
 ### Run all unit tests
 
+The `hack/test-go.sh` script is the entrypoint for running the unit tests with
```

Comment:
It's arguable either way, because english.  It runs the tests with the GOPATH set up correctly -- and allows you to run them without having GOPATH set up correctly.